benchmark/src/main/content/bin/kcb.sh

```diff
@@ -34,6 +34,8 @@ JAVA_OPTS="${JAVA_OPTS} -Xmx1G -XX:+HeapDumpOnOutOfMemoryError"
 DEBUG_MODE="${DEBUG:-false}"
 DEBUG_PORT="${DEBUG_PORT:-8787}"
 
+CHAOS_MODE="${CHAOS_MODE:-false}"
```

Comment:
For distributed runners, this mode is not very usable as we don't want to run the chaos script from all runners, but only once.

We can move the trigger back to the workflow (the log directory can be obtained also there) and then we can print the info from the pod at the end of kcb script from another workflow step. This way we would also get rid of coupling of kcb script with Openshift. Any thoughts @kami619 @ahus1 @tkyjovsk?

Reply:
thanks for the review comments everyone. given the majority feedback leans towards modularization of this, I shall work with Michal tomorrow to come up with a better design and implement that.

Reply:
in order to keep the scope intact and not let the design decisions take up more time for this PR to get merged. we are going to merge this as is created a [ticket](https://github.com/keycloak/keycloak-benchmark/issues/405) to tackle the modularization of script and solving the puzzle with distributed load tests in mind as the next increment.